### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
- Completed layout content of the tech cards.

## Screenshots
image

image

## Trello Card
https://trello.com/c/rVK66sYw

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -8,20 +8,127 @@
...
-                          <h3 class="m-bottom-1">ReactJS</h3>
-                          <span class="tech-release m-bottom-1">Release: 2013</span>
-                          <p class="tech-description">React is a declarative, component-based JavaScript libar for building user interfaces</p>
-                      </div>
-                  </article>
-            </div>
-          </section>
-      </main>
+    <main class="main-project">
+      <section class="section-project l-container">
```
Comment (CarlosHWolox) [inline maquetado/index.html]: I don't understand what `l-container` means, i would suggest a more declarative name.

File: maquetado/index.html [hunk 2]
```diff
@@ -8,20 +8,127 @@
...
+        <h1 class="main-title m-bottom-1">Front-End Tools</h1>
+        <div class="techs-container">
+          <div class="row">
```
Comment (CarlosHWolox) [inline maquetado/index.html]: Is this div really necessary? Could not be this styles applied to techs-container class instead?

File: maquetado/index.html [hunk 3]
```diff
@@ -8,20 +8,127 @@
...
+                </p>
+              </div>
+            </article>
+          </div>
+        </div>
+      </section>
+      <section class="section-project l-container">
+        <h1 class="main-title m-bottom-1">Back-End Tools</h1>
+        <div class="techs-container">
+          <div class="row">
```
Comment (CarlosHWolox) [inline maquetado/index.html]: same here

File: maquetado/index.html [hunk 4]
```diff
@@ -9,15 +9,11 @@
...
         <h1 class="main-title m-bottom-1">Front-End Tools</h1>
-        <div class="tech-box">
+        <div class="techs-container row">
           <article class="tech">
-            <img
-              src="../assets/react_icon.svg"
-              alt="react-logo"
-              class="tech-logo"
-            />
+            <img src="../assets/react_icon.svg" alt="" class="tech-logo" />
```
Comment (alejobh) [inline maquetado/index.html]: put something related to te tech or the tech title in `alt` attr

**Pull request merged**